TCAT/Controllers/HomeOptionsCardViewController.swift

```diff
@@ -119,13 +118,19 @@ class HomeOptionsCardViewController: UIViewController {
         searchBar.placeholder = Constants.General.searchPlaceholder
         searchBar.delegate = self
         searchBar.searchBarStyle = .default
-        let textFieldInsideSearchBar = searchBar.value(forKey: "searchField") as? UITextField
-        textFieldInsideSearchBar?.backgroundColor = Colors.white
+        searchBar.returnKeyType = .search
         searchBar.barTintColor = .white
         searchBar.layer.borderColor = UIColor.white.cgColor
         searchBar.layer.borderWidth = 1
+
+        if let textFieldInsideSearchBar = searchBar.value(forKey: "searchField") as? UITextField,
+            let searchView = textFieldInsideSearchBar.leftView as? UIImageView {
+            textFieldInsideSearchBar.backgroundColor = Colors.white
+            searchView.image = #imageLiteral(resourceName: "search-large")
+        }
+
         // Add horizontal offset so that placeholder text is aligned with bus stop names and all stops
-        searchBar.searchTextPositionAdjustment = UIOffset(horizontal: 8, vertical: 0)
+        searchBar.searchTextPositionAdjustment = UIOffset.init(horizontal: 8, vertical: 0)
```

Comment:
Any specific reason for adding this? No big deal

Reply:
Good catch not sure y I did that